pkg/api/kubegraph/edges.go

```diff
@@ -0,0 +1,46 @@
+package kubegraph
+
+import (
+	"github.com/gonum/graph"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+
+	osgraph "github.com/openshift/origin/pkg/api/graph"
+	kubegraph "github.com/openshift/origin/pkg/api/kubegraph/nodes"
+	deploygraph "github.com/openshift/origin/pkg/deploy/graph/nodes"
+)
+
+const (
+	ExposedThroughServiceEdgeKind = "ExposedThroughService"
+)
+
+// AddFullfillingDeploymentConfigEdges ensures that a directed edge exists between all deployment configs and the
+// services that expose them (via label selectors).
+func AddFullfillingDeploymentConfigEdges(g osgraph.MutableUniqueGraph, node *kubegraph.ServiceNode) *kubegraph.ServiceNode {
```

Comment:
It depends on what you're using the edge for, I guess. If you want to know what services are pointing at your DC, then dc -> svc makes sense to me. And that's why I asked the question - since this code currently is creating the dc -> svc relationship, it seems like you'd want to loop through all dcs to set up these edges. And move the code to deploy.